Database/CreateProductTable.sql

```diff
@@ -7,6 +7,6 @@ CREATE TABLE CoroNacessitiesDB.dbo.Product(
 [Price] DECIMAL(10,2) NOT NULL,
 [ProductStatusID] INT NOT NULL,
 [TypeID] INT NOT NULL,
-[ProductImage] VARCHAR(MAX),
+[ProductImage] VARBINARY(MAX),
```

Comment:
This breaks the product table sql insert queries.

Reply:
Type mismatch now.

Reply:
Is it actually giving errors or are the data entries just not succeeding?

Reply:
It gives errors if you run it in micosoft sql studio, but not if you run it in console. In console it looks like it just skips over them.

Reply:
You can put whatever temp values in there for now, just as long as they work.

Reply:
So would you prefer I change this back to a string or that I change the data entries?

Reply:
Change the data entries. The backend is expecting varbinary.

Reply:
Trying to figure out how to store images as blobs now